app/build.gradle

```diff
@@ -371,15 +370,16 @@ dependencies {
     testImplementation 'org.robolectric:robolectric:4.4'
     testImplementation 'org.robolectric:shadows-multidex:4.4'
 
-    implementation 'com.github.bumptech.glide:compose:1.0.0-alpha.5'
-    implementation 'androidx.compose.ui:ui:1.5.2'
-    implementation 'androidx.compose.ui:ui-tooling:1.5.2'
+    implementation 'com.github.bumptech.glide:compose:1.0.0-beta01'
+
+    implementation "androidx.compose.ui:ui:$androidxComposeVersion"
```

Comment:
If we are cleaning up the imports, it might be worth switching to the BOM declaration:
https://developer.android.com/develop/ui/compose/bom

Reply:
Done - see: `app/build.gradle`